packages/context-entries/lib/Keys.ts

```diff
@@ -185,6 +185,10 @@ export const KeysInitQuery = {
    * A boolean value denoting whether caching is disabled or not.
    */
   noCache: new ActionContextKey<boolean>('@comunica/actor-init-query:noCache'),
+  /**
+   * A boolean value denoting whether results should be deduplicated or not.
+   */
+  distinct: new ActionContextKey<boolean>('@comunica/actor-init-query:distinct'),
```

Comment:
Could you also add a context shortcut here? https://github.com/comunica/comunica/blob/master/packages/actor-context-preprocess-convert-shortcuts/lib/ActorContextPreprocessConvertShortcuts.ts#L43-L71